### PR TITLE
Update learn-about-office-365-germany.md

### PR DESCRIPTION
**Pull request by msbemba**
removed the sign-in link after discussing with Loreen.

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6834

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/admin-overview/learn-about-office-365-germany.md [hunk 1]
```diff
@@ -86,7 +86,6 @@ The following services and features are available in some [Office 365 Germany su
 
 |Item|Information|
 |-----|-----|
-|Sign in portal  <br/> |[https://portal.office.de](https://portal.office.de)  <br/> |
 |Subscriptions, billing, and technical support  <br/> |Germany-based support available in German and English For details, see [Contact support for business products - Admin Help](../../business-video/get-help-support.md).  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/admin-overview/learn-about-office-365-germany.md]:
```suggestion
|Subscriptions, billing, and technical support  <br/> |Germany-based support available in German and English. For details, see [Contact support for business products - Admin Help](../../business-video/get-help-support.md).  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**